src/components/MemberProfileView.tsx

```diff
@@ -1,130 +1,353 @@
+import React from 'react';
 import {
   Box,
   Text,
   VStack,
+  HStack,
   Avatar,
   Link,
+  Badge,
   useColorModeValue,
   SimpleGrid,
   Divider,
   Stack,
-  Badge,
+  Icon,
+  Flex,
+  Tooltip,
+  Button,
+  useClipboard,
+  useToast,
 } from '@chakra-ui/react';
-import { Member } from '../types';
+import {
+  FaGithub,
+  FaLinkedin,
+  FaExternalLinkAlt,
+  FaDiscord,
+  FaCode,
+  FaCalendar,
+  FaMapMarkerAlt,
+  FaGraduationCap,
+  FaEnvelope,
+  FaClock,
+  FaIdBadge,
+  FaUsers,
+  FaCopy,
+} from 'react-icons/fa';
+
+interface SocialField {
+  username: string;
+}
+
+interface Member {
```

Comment:
Why are we redeclaring the member type here?

Reply:
oops I was just giving copilot the context